website/project/signals.py

```diff
@@ -2,6 +2,7 @@
 
 signals = blinker.Namespace()
 comment_added = signals.signal('comment-added')
+mention_added = signals.signal('mention-added')
```

Comment:
Eep. Generally not a huge fan of signals, but it seems to be the established pattern for the comments code, so I guess it makes sense to use that on this PR.